go/client/cmd_ctl_start_osx.go

```diff
@@ -6,46 +6,93 @@
 package client
 
 import (
+	"fmt"
+	"strings"
+	"time"
+
 	"github.com/keybase/cli"
 	"github.com/keybase/client/go/install"
+	"github.com/keybase/client/go/launchd"
 	"github.com/keybase/client/go/libcmdline"
 	"github.com/keybase/client/go/libkb"
 )
 
+// NewCmdCtlStart constructs ctl start command
 func NewCmdCtlStart(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 	return cli.Command{
 		Name:  "start",
-		Usage: "Start the keybase service",
+		Usage: "Start the app and services",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "exclude",
+				Usage: fmt.Sprintf("Start all except excluded components, comma separated. Specify %v.", availableCtlComponents),
+			},
+		},
 		Action: func(c *cli.Context) {
-			cl.ChooseCommand(NewCmdCtlStartRunner(g), "start", c)
+			cl.ChooseCommand(newCmdCtlStart(g), "start", c)
 			cl.SetForkCmd(libcmdline.NoFork)
 			cl.SetLogForward(libcmdline.LogForwardNone)
 			cl.SetNoStandalone()
 		},
 	}
 }
 
-type CmdCtlStart struct {
+type cmdCtlStart struct {
 	libkb.Contextified
+	components map[string]bool
 }
 
-func NewCmdCtlStartRunner(g *libkb.GlobalContext) *CmdCtlStart {
-	return &CmdCtlStart{libkb.NewContextified(g)}
+func newCmdCtlStart(g *libkb.GlobalContext) *cmdCtlStart {
+	return &cmdCtlStart{
+		Contextified: libkb.NewContextified(g),
+		components:   defaultCtlComponents,
+	}
 }
 
-func (s *CmdCtlStart) ParseArgv(ctx *cli.Context) error {
+func (s *cmdCtlStart) ParseArgv(ctx *cli.Context) error {
```

Comment:
this is going to be the same in `start`, `stop`, `restart`...use common function.